bundle/src/main/java/com/adobe/acs/commons/workflow/process/impl/ReplicateWithOptionsWorkflowProcess.java

```diff
@@ -188,5 +280,57 @@ public boolean isThrottle() {
             return throttle;
         }
 
+        public ReplicationOptions getReplicationOptionsDispatcher(Resource content) {
+            ReplicationOptions replicationOptionsDispatcher = new ReplicationOptions();
+            replicationOptionsDispatcher.setSynchronous(
+                replicationOptionsDispatcher.isSynchronous()
```

Comment:
I don't get this line here? Is source and target really both the new object `replicationOptionsDispatcher`?